relay/sources/rudder-pkg/lib/rudder-pkg/rpkg.py

```diff
@@ -50,7 +52,7 @@ def __ge__(self, other):
 class PluginVersion:
     def __init__(self, pluginLongVersion):
         match = re.search(
-            r'(?P<rudderVersion>(?P<rudderMajor>[0-9]+\.[0-9]+)(\.[0-9]+(~(beta|rc)[0-9]+)?)?)-(?P<pluginShortVersion>[0-9]+\.[0-9]+)(-(?P<mode>[a-zA-Z]+))?',
+            r'(?P<rudderVersion>(?P<rudderMajor>[0-9]+\.[0-9]+)(\.[0-9]+(~(alpha|beta|rc)[0-9]+)?)?)-(?P<pluginShortVersion>[0-9]+\.[0-9]+)(-(?P<mode>[a-zA-Z]+))?',
```

Comment:
We should update the version comparison tests to match the new plugin version format.

Reply:
damn i forgot one, good catch

Reply:
I updated some tests! Is is ok for you now @Fdall ?